leaderless/node/app/acceptor.py

```diff
@@ -0,0 +1,19 @@
+class Acceptor:
+    def __init__(self):
+        self.promised_id = -1
+        self.accepted_id = None
+        self.accepted_val = None
+
+    def handle_prepare(self, propose_id):
+        if self.promised_id >= propose_id:
+            self.promised_id = propose_id
+            return self.accepted_id, self.accepted_val
+        return None
```

Comment:
When acceptor sees that it should ignore the message, make it send back self.promised_id, so that proposer knows what is the required minimum id.

Reply:
i.e. "Nack" response from wikipedia article.